src/components/TextInput/BaseTextInput/index.native.tsx

```diff
@@ -272,6 +272,7 @@ function BaseTextInput(
                 <PressableWithoutFeedback
                     onPress={onPress}
                     tabIndex={-1}
+                    role={inputProps.role}
```

Comment:
I passed down the `role` prop here for the native BaseTextInput for good measure as it can't do any harm and by doing this we're aligned on both native and the rest ✌